src/Fantomas.Cmd/Program.fs

```diff
@@ -241,8 +247,35 @@ let main _args =
 
     ArgParser.Parse(options, handleInput, "Fantomas <input_path>")
 
+    let loadConfig path =
```

Comment:
Is a Result option really the correct type here? This seems like a good case for a DU with ConfigNotLoad, ConfigInvalid and Config.

Reply:
I like this